client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -35,6 +36,7 @@ export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) =>
 		onContentOnlyClick,
 		ctaText,
 		subTitleText,
+		showTitleAndSubTitle = true,
```

Comment:
Suggestion:  Invert the logic, because on react and HTML the a lack of attribute usually means false.

Example:
 -- Enabled label <label>
 -- Disabled label <label disabled>

On this PR my suggestion is 
-- Default <Component>
-- New usage <Component hideTitleAndSubtitle>

it also prevent us to have issues if we decided to have the title and subtitle hidden by default.